docs/v0.6/en/greptimecloud/integrations/influxdb.md

```diff
@@ -11,6 +11,8 @@ Please refer to [InfluxDB client](https://docs.greptime.com/user-guide/clients/i
 - Username: `<username>`
 - Password: `<password>`
 
+For example, the following example writes a data point to the `system_metrics` table:
+
 ```sh
 curl -i 'https://<host>/v1/influxdb/write?db=<dbname>&u=<username>&p=<password>' \
 --data-binary 'system_metrics,host=host1,idc=idc_a cpu_util=11.8,memory_util=10.3,disk_util=10.3 1667446797450000000'
```

Comment:
I think we can add an example of using an influxdb client for this, using any programming language, just to demo the configuration.

Reply:
researching and practicing, it will take some time...